doc/usr/source/9_other/10_inductive_validity_core.rst

```diff
@@ -82,17 +85,11 @@ We obtain the following inductive validity core:
 Computing all Inductive Validity Cores
 --------------------------------------
 
-If we want to compute ALL the minimal inductive validity cores, we must select the implementation ``UMIVC``:
+If we want to compute ALL the minimal inductive validity cores, we can use the following flags:
 
 .. code-block:: none
 
-  kind2 <lustre_file> --ivc true --ivc_impl UMIVC
+  kind2 <lustre_file> --ivc true --ivc_all true --ivc_must_set true
 
-The ``UMIVC`` implementation does not terminate until all minimal inductive validity cores have been found,
-but when an inductive validity core is found, it is directly printed (no need to wait for the algorithm to terminate).
-
-``UMIVC`` is composed of two phases: the first one reduces the search space by computing some Maximal Unsafe Abstractions,
-and the second one search for Minimal Inductive Validity Cores.
-A parameter ``k`` between 0 and infinity determines how long the first phase will be.
-More precisely, the first phase computes all the Maximal Unsafe Abstractions of cardinality up to ``k``.
-``k`` can be set using the option ``--ivc_umivc_k``, with the value -1 for infinity.
+* ``--ivc_all true``: specify that we want to compute all the IVCs
+* ``--ivc_must_set true``: indicate that the MUST set should be computed first. In general, when ``--ivc_all`` is ``true``, it is more efficient to enable this option
```

Comment:
When ``--ivc_all`` is ``true``, the MUST set should be always computed first. The difference is, if ``--ivc_must_set`` is ``true``, then the MUST set is printed in the output. Otherwise, it is only used internally.